database/seeds/AssignDefaultRoleToUserSeeder.php

```diff
@@ -13,11 +14,23 @@ class AssignDefaultRoleToUserSeeder extends Seeder
      */
     public function run()
     {
+        /** @var Role $admin */
+        $admin = Role::whereName('admin')->first();
+        /** @var Role $developer */
+        $developer = Role::whereName('developer')->first();
+        /** @var Role $team_member */
+        $team_member = Role::whereName('team_member')->first();
+
+        $permissions = Permission::all();
+        $admin->givePermissionTo($permissions);
+        $developer->givePermissionTo($permissions);
+
+        $permissions = Permission::whereIn('name', ['manage_tags', 'manage_activities', 'manage_reports', 'manage_all_tasks'])->get();
+        $team_member->givePermissionTo($permissions);
+
         $roleIds = [];
```

Comment:
all below this lines are not needed remove it, because it will give admin roles to all users.